pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -235,7 +235,7 @@ self: super: {
   aws = dontCheck super.aws;                            # needs aws credentials
   aws-kinesis = dontCheck super.aws-kinesis;            # needs aws credentials for testing
   binary-protocol = dontCheck super.binary-protocol;    # http://hydra.cryp.to/build/499749/log/raw
-  binary-search = dontCheck super.binary-search;
+  binary-search = dontCheck (appendPatch super.binary-search ./patches/binary-search-1.0.0.3-ghc-8.8.patch);
```

Comment:
Is there anyway you can pull this patch from upstream?  Normally we try to avoid carrying around patches here in nipxkgs if possible.

Reply:
Ping?